charts/nri-prometheus/templates/configmap.yaml

```diff
@@ -8,6 +8,8 @@ apiVersion: v1
 data:
   config.yaml: |
     cluster_name: {{ include "nri-prometheus.cluster" . }}
-{{- if .Values.config }}
-{{ toYaml .Values.config | indent 4 -}}
-{{ end }}
+{{- if (include "nri-prometheus.lowDataMode" .) }}
+    {{- .Files.Get "static/lowdatamodedefaults.yaml" | indent 4 -}}
```

Comment:
If `lowDataMode=true` how a customer could change any of the injected parameters?

Do we need to inject a whole new static config or only to enforce:
``` yaml
transformations:
  - description: "Low data mode defaults"
    ignore_metrics:
      # Ignore the following metrics.
      # These metrics are already collected by the New Relic Kubernetes Integration.
      - prefixes:
        - kube_
        - container_
        - machine_
        - cadvisor_
```

Reply:
I modified the PR to overwrite only the transformation list